time-clock/db/migrate/20211222000109_create_employees.rb

```diff
@@ -0,0 +1,21 @@
+class CreateEmployees < ActiveRecord::Migration[6.1]
+  def change
+    create_table :employees do |t|
+      t.string :name
+      t.string :position
+      t.integer :branch_id
```

Comment:
I think the `belongs_to` "branch" below statements covers the use case for this column, right?